k9mail/src/main/java/com/fsck/k9/activity/setup/AccountSettings.java

```diff
@@ -1029,6 +1044,17 @@ private void updateRemoteSearchLimit(String maxResults) {
         }
     }
 
+    private void goToPreferenceScreen(PreferenceScreen preference) {
```

Comment:
This makes me sad. Hopefully this will be easier once we rework the settings activities.